src/components/Button.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './Button.css';
 
-function Button() {
+function Button({ buttonName }) {
   return (
     <div className="Button" id="button">
-      <p> Calculator App: Hello world!</p>
+      <p>
+        { buttonName}
+      </p>
     </div>
   );
 }
 
+Button.defaultProps = {
+  buttonName: 'defaultName',
+};
```

Comment:
- [ ] There is no need to declare default props for the `buttonName` props. Instead, we need to declare this prop as a required prop. If we think about it, in the context of a calculator we want to always inform the button name.